### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by jpizzla** [Ajarmar/universal-pokemon-randomizer-zx#553]
Has there been any updates on why I get this error when trying to open the launcher. I have tried moving the files around to where I may have more permissions but nothing is working :(

<img width="598" alt="Screen Shot 2022-10-07 at 11 02 14 am" src="https://user-images.githubusercontent.com/115196277/194439433-37df091e-f711-4ede-a5e6-c31c37db303c.png">

**Comment by tom-overton** [comment]
Is launcher_MAC.command still in the same folder as PokeRandoZX.jar? They need to be in the same folder to work.

**Comment by jpizzla** [comment]
> Is launcher_MAC.command still in the same folder as PokeRandoZX.jar? They need to be in the same folder to work.

Yes they are :( please help me ill paypal you $20 if you can pllsssss

**Comment by tom-overton** [comment]
There's no need to pay us. Let's try to solve this, because you aren't the only one who has run into this, and it would be nice if we could fix it for everyone.

If you double-click on `PokeRandoZX.jar`, does it open the randomizer? It will complain about not being able to randomize the 3DS games, but let's ignore that for now. If that *doesn't* work, then something is up with your Java installation; refer to our [About Java](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/wiki/About-Java) wiki page for tips to solve it.

If double-clicking *does* work, then let's try this out and see if it works:
- Open the Terminal. You can do this by pressing Spacebar + Command, typing "Terminal" and pressing Enter.
- Type `java -jar ` into the Terminal, but *don't* press Enter. There's a space after the `-jar` too; this is important.
- Find `PokeRandoZX.jar` in the randomizer folder. Drag-and-drop it onto your Terminal window. This should automatically update the command to look something like `java -jar path/to/PokeRandoZX.jar`
- Press Enter to run the command.

Does this work? What we're doing here is basically doing what the launcher is *trying* to do, but we're just doing it manually. If this works, then there's something wrong with the launcher.
